### DISCUSSION TITLE
How to add our own blogs

### DISCUSSION THREAD

**Discussion by Brshank** [sharadcodes/jekyll-theme-serial-programmer#15] [Q&A / Get Help] [answered]
How to add your own blogs and link it .
i am currently having problems in the code to understand

**Comment by sharadcodes** [comment, marked as answer]
Add them inside `all_collections/_posts` folder, as for linking that would be done automatically when build process is  completed

**Reply by sharadcodes** [reply]
No, any number of posts you want
